services/content/content_test.go

```diff
@@ -36,6 +36,46 @@ func createContentService(t *testing.T) (*cs.ContentService, *rr.Server) {
 				"name": "name",
 				"label": "Name",
 				"sortable": true
+			},
+			{
+				"type": "relation",
+				"name": "tags",
+				"label": "Tags",
+				"optional": true,
+				"sortable": true,
+				"relation": {
+					"schema": "tag",
+					"field": "blogs",
+					"type": "o2o",
+					"owner": true,
+					"fk_columns": null,
+					"optional": false
+				}
+			}
+		]
+	}`)
+	utils.WriteFile(schemaDir+"/tag.json", `{
+		"name": "tag",
+		"namespace": "tags",
+		"label_field": "name",
+		"fields": [
+			{
+				"type": "string",
+				"name": "name",
+				"label": "Name",
+				"sortable": true
+			},
+			{
+				"type": "relation",
+				"name": "blogs",
+				"label": "Blogs",
+				"relation": {
+					"schema": "blog",
+					"field": "tags",
+					"type": "o2o",
+					"fk_columns": null,
+					"optional": false
+				}
```

Comment:
I think we should test with o2m relation, and the `tag.blogs` should be the relation owner.
```suggestion
			{
				"type": "relation",
				"name": "tags",
				"label": "Tags",
				"optional": true,
				"sortable": true,
				"relation": {
					"schema": "tag",
					"field": "blogs",
					"type": "o2m",
					"fk_columns": null,
					"optional": false
				}
			}
		]
	}`)
	utils.WriteFile(schemaDir+"/tag.json", `{
		"name": "tag",
		"namespace": "tags",
		"label_field": "name",
		"fields": [
			{
				"type": "string",
				"name": "name",
				"label": "Name",
				"sortable": true
			},
			{
				"type": "relation",
				"name": "blogs",
				"label": "Blogs",
				"relation": {
					"schema": "blog",
					"field": "tags",
					"type": "o2m",
					"owner": true,
					"fk_columns": null,
					"optional": false
				}
```